### DISCUSSION TITLE
Many achievements seem to not be showing even though I know I've accomplished them.

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [community/community#39537] [Other Feature Feedback, Questions, & Ideas] [answered]
Many achievements listed in https://github.com/Schweinepriester/github-profile-achievements do not seem to appear on my profile, or other's profiles who I know also accomplished said tasks, and I can't seem to figure out why.

I've tried logging in and out, clearing cache and using a different browser (my default is FireFox Developer Edition).

Are other's having this issue? I know they are still in beta but I'm hoping to have them displayed. My profile settings is set to have them on as well.

**Comment by radar-blip** [comment, marked as answer]
You have 3 showing. 
If you have others, try clicking on the `Achievements` link under profile picture and unhide them by clicking the eye icon.

![achievy](https://user-images.githubusercontent.com/39196767/202876445-5bd67e40-487c-41c5-8962-1e03ab234de0.png)  

![showoff](https://user-images.githubusercontent.com/39196767/202876556-79983c4a-13bb-4c22-9047-a2b11b2edb87.png)

**Comment by jfullstackdev** [comment]
did you do all of the requirements ? if not, you will not be getting that

**Comment by chris-c-thomas** [comment]
Okay, overall I think I was just not understanding a few things correctly. Thanks for the help everyone.
